telegram/ext/jobqueue.py

```diff
@@ -226,10 +244,10 @@ def run_daily(self, callback, time, days=Days.EVERY_DAY, context=None, name=None
                   interval=datetime.timedelta(days=1),
                   repeat=True,
                   days=days,
-                  tzinfo=time.tzinfo,
                   context=context,
                   name=name,
-                  job_queue=self)
+                  job_queue=self,
+                  tzinfo=time.tzinfo)
```

Comment:
There is no real change here, only confuses the reader or possible code merges.
Please revert.